Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -5546,6 +5546,7 @@ void CClientGame::ResetMapInfo()
     g_pGame->GetWeather()->ResetWaterFog();
     g_pGame->GetWeather()->ResetSandstorm();
     g_pGame->GetWeather()->ResetRainbow();
+    g_pGame->GetSettings()->SetVolumetricShadowsEnabled(g_pGame->GetSettings()->GetVolumetricShadowsEnabledByVideoSetting());
```

Comment:
Since both methods(`SetVolumetricShadowsEnabled` and `GetVolumetricShadowsEnabledByVideoSetting`) are from CSettingsSA it would be better to create the specialized method `ResetVolumetricShadows()`.

Reply:
but what does mean ResetVolumetricShadows it will reset to the video setting or calling SetVolumetricShadowsEnabled  to false?

Reply:
I mean `ResetVolumetricShadows` is equivalent to `g_pGame->GetSettings()->SetVolumetricShadowsEnabled(g_pGame->GetSettings()->GetVolumetricShadowsEnabledByVideoSetting());`